index.d.ts

```diff
@@ -120,11 +120,11 @@ export interface AssertContext {
 	/**
 	 * Assert that value is deep equal to expected.
 	 */
-	same<U>(value: U, expected: U, message?: string): void;
+	deepEqual<U>(value: U, expected: U, message?: string): void;
```

Comment:
That's probably the only way to handle it. Looks like TS has a two year old issue on the topic:
https://github.com/Microsoft/TypeScript/issues/390